app/ParserTests/ParserTests.hs

```diff
@@ -6,12 +6,77 @@ Module containing test cases for Requirement Parsers.
 -}
 
 module ParserTests.ParserTests
-( reqTestSuite ) where
+(  reqTestSuite  ) where
 
-import Test.HUnit
+import qualified Text.Parsec as Parsec
+import Text.Parsec.String (Parser)
+import Database.Requirement
+import WebParsing.ReqParser
+import Test.HUnit ( assertEqual, Test(..) )
 
-test1 :: Test
-test1 = TestCase (assertEqual "TEST PASS" ("True") ("True"))
+-- Function to facilitate test case creation given a string, Req tuple
+createTest :: (Eq a, Show a) => Parser a -> (String, a) -> Test
```

Comment:
You are right to want to remove the code duplication below. Change this function so that its second parameter has type `[(String, a)]` so that you can pass the whole list in at once.